src/com/aca/homework/week4/snake/Snake.java

```diff
@@ -0,0 +1,146 @@
+package com.aca.homework.week4.snake;
+
+import java.util.Random;
+import java.util.Scanner;
+
+public class Snake {
+    private char[][] board;
```

Comment:
it is weird that snake a board or Snake class has a snake matrix. Please take a lot of attention to namings